packages/vanilla-components/src/components/stats-container/stats-container.tsx

```diff
@@ -87,12 +88,12 @@ export class StatsContainer {
   }
 
   getRewardBalance(statVariables) {
-    const { type, unit, valuetype } = statVariables;
+    const { type, unit, valuetype, global } = statVariables;
     // passing in CASH/USD as CASH_USD into this webcomponent so pathToRegexp doesn't split it and it stays all in unit.
     // this converts the _ back to / so that it matches the unit from the backend
     // possibly change the unit in the backend to avoid this?
     const newUnit = unit.replace(/_/, "/")
-    const rewardBalance = this.stats['rewardBalances'].find(rb => rb.type === type && rb.unit === newUnit);
+    const rewardBalance = this.stats[global === "global" ? 'rewardBalanceGlobal' : 'rewardBalance'].find(rb => rb.type === type && rb.unit === newUnit);
```

Comment:
```suggestion
    const rewardBalance = this.stats[global === "global" ? "rewardBalanceGlobal" : "rewardBalance"].find(rb => rb.type === type && rb.unit === newUnit);
```

Or , you could probably make an assumption that if `global` exists, then it's global.

```suggestion
    const rewardBalance = this.stats[`rewardBalance${global && "Global"} `].find(rb => rb.type === type && rb.unit === newUnit);
```